tokio-timer/src/delay_queue.rs

```diff
@@ -171,7 +171,7 @@ pub struct Expired<T> {
 /// documentation for more details.
 ///
 /// [`DelayQueue`]: struct.DelayQueue.html
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Why isn't it `Copy`?

Reply:
According to https://doc.rust-lang.org/std/marker/trait.Copy.html#when-should-my-type-be-copy 

> Generally speaking, if your type can implement Copy, it should.

Reply:
Word of @carllerche on gitter was to do no more than absolutely necessary.

Reply:
I usually avoid doing so for forwards compat. If it derives `Copy`, this means it can never *not* be copy.

That said, this is a low risk case. It is doubtful the struct contents will ever change. So, I could go either way here.

Reply:
I really can't imagine a reason it would need not to be `Copy`, but I don't really mind either way myself.

Reply:
I understand Carl's concerns.

> I really can't imagine a reason it would ever need not to be Copy

If there is a new field inside `Key` that will be `!Copy`

Reply:
I meant that it's not clear why that might ever happen.

Reply:
@Ralith If you want to add a `Copy` derive, I will add it. I'm also going to merge this now to avoid blocking further.